akka-docs/src/main/paradox/stream/operators/Source/repeat.md

```diff
@@ -1,20 +1,33 @@
 # Source.repeat
 
-Stream a single object repeatedly
+Stream a single object repeatedly.
 
 @ref[Source operators](../index.md#source-operators)
 
-@@@div { .group-scala }
-
 ## Signature
 
-@@signature [Source.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala) { #repeat }
-
-@@@
+@apidoc[Source.repeat](Source$) { scala="#repeat[T](element:T):akka.stream.scaladsl.Source[T,akka.NotUsed]" java="#repeat(T)" }
 
 ## Description
 
-Stream a single object repeatedly
+This source emits a single element repeatedly. It never completes, so the stream will in most cases
+be limited by other means.
+
+See also:
+
+* @ref:[`single`](single.md) Stream a single object once.
+* @ref:[`tick`](tick.md) A periodical repetition of an arbitrary object.
+* @ref:[`cycle`](cycle.md) Stream iterator in cycled manner.
+
+## Example
+
```

Comment:
"In this sample we make the stream finite by combining with `take` making it complete after 17 elements:"